File: test/hello_codex_test.js
function summarize(numbers) {
  const total = numbers.reduce((sum, value) => sum + value, 0);
  const average = total / numbers.length;

  return {
    count: numbers.length,
    total,
    average,
    minimum: Math.min(...numbers),
    maximum: Math.max(...numbers),
  };
}

function main() {
  const sample = [3, 7, 11, 18, 23];
  const stats = summarize(sample);

  console.log("Codex code test succeeded.");
  console.log(`Input numbers: ${sample.join(", ")}`);
  console.log(
    `Summary: count=${stats.count}, total=${stats.total}, ` +
      `average=${stats.average.toFixed(2)}, min=${stats.minimum}, max=${stats.maximum}`
  );
}

main();

File: test/hello_codex_test.py
def summarize(numbers):
    total = sum(numbers)
    average = total / len(numbers)
    return {
        "count": len(numbers),
        "total": total,
        "average": average,
        "minimum": min(numbers),
        "maximum": max(numbers),
    }


def main():
    sample = [3, 7, 11, 18, 23]
    stats = summarize(sample)

    print("Codex code test succeeded.")
    print(f"Input numbers: {sample}")
    print(
        "Summary: "
        f"count={stats['count']}, "
        f"total={stats['total']}, "
        f"average={stats['average']:.2f}, "
        f"min={stats['minimum']}, "
        f"max={stats['maximum']}"
    )


if __name__ == "__main__":
    main()
